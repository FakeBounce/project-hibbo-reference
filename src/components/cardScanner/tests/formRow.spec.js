import React from 'react';
import renderer from 'react-test-renderer';

import FormRow from '../components/FormRow';

describe('FormRow', () => {
  const fn = jest.fn();
  const component = renderer.create(
    <FormRow
      labelName="Some label"
      value=""
      label="Some label"
      name=""
      onChange={fn}
      validate={fn}
      error={false}
      width={300}
      maxLength={3}
    />,
  ).toJSON();

  it('should render the component', () => {
    expect(component).toMatchSnapshot();
  });
});
