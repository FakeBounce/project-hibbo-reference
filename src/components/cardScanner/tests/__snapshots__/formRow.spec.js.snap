// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormRow should render the component 1`] = `
<View>
  <View
    style={undefined}
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      disabled={false}
      ellipsizeMode="tail"
      style={undefined}
    >
      Some label
    </Text>
  </View>
  <View
    style={
      Array [
        undefined,
        Object {
          "width": 300,
        },
      ]
    }
  >
    <TextInput
      autoCapitalize="words"
      autoCorrect={false}
      autoFocus={false}
      keyboardType="numeric"
      maxLength={3}
      onChangeText={[Function]}
      onSubmitEditing={[Function]}
      placeholder="Some label"
      placeholderTextColor="#979797"
      returnKeyType="next"
      style={
        Array [
          undefined,
          Object {
            "color": "#413D3D",
            "width": 300,
          },
        ]
      }
      underlineColorAndroid="transparent"
      value=""
    />
  </View>
</View>
`;
